### PR TITLE
Added Coffee machine initial project

### PR DESCRIPTION
**Pull request by nsimonyan**

### CODE REVIEW

File: Coffee-machine/src/Machine.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Machine {
+    public static void main(String[] args) {
+        System.out.println("Hello Coffe");
```
Comment (ArmineZak) [inline Coffee-machine/src/Machine.java:3]: Hello team!!!!

Comment (GayaneGulanyan) [inline Coffee-machine/src/Machine.java:3]: @Arayikkhachatryan please change Coffe to Coffee

Comment (Arayikkhachatryan) [reply Coffee-machine/src/Machine.java:3]: Hii Armine Jan

Comment (Arayikkhachatryan) [reply Coffee-machine/src/Machine.java]: Fixed
Done

**Pull request merged**